gravitee-apim-gateway/gravitee-apim-gateway-services/gravitee-apim-gateway-services-healthcheck/src/main/java/io/gravitee/gateway/services/healthcheck/EndpointHealthcheckResolver.java

```diff
@@ -141,9 +141,19 @@ public List<EndpointRule> resolve(Api api) {
                         : endpoint.getHealthCheck();
                     // The following has to be managed by the connector-api
                     if (endpoint.getType().equalsIgnoreCase("grpc")) {
-                        return new GrpcEndpointRule(api.getId(), endpoint, healthcheck, systemProxyOptions);
+                        return new GrpcEndpointRule(
+                            (io.gravitee.gateway.handlers.api.definition.Api) api,
```

Comment:
I'm wondering if this cast could be handled at an upper level (eg: `EndpointHealthcheckVerticle`) instead ?

Reply:
Done 👍🏼